### PR TITLE
Add filing a bug page

### PR DESCRIPTION
**Pull request by pvanderknyff**
For TFS1172882

Not entirely sure the messaging we want for the expectations when people file a bug, open to suggestions on that. Hopefully this will cut down on us needing to constantly ask for logs.

### CODE REVIEW

File: docs/bug-artifacts.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Filing a Bug with the SDK
+---
+
+If you encounter a bug with the Tableau Connector SDk, please file it as a github issue. We will take a look at it and respond as soon as we are able.
```
Comment (tsjoblad) [inline docs/bug-artifacts.md:5]: SDK

File: docs/bug-artifacts.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+When filing an bug on github, attaching the following to the issue:
+- A readme file that contains the following (can be in the issue itself):
+    - Your name and company
+    - A short description of the bug
+    - Repro steps with screenshots
+    - The product(s) it appears in
+    - The exact version of the Tableau products you are using (ie Desktop 2020.3.1, Server 2020.2.2, TDVT 2.1.1, etc), as relevant to the bug
+    - The operating system you are using
+- Connector Files. Can be either a folder with the loose connector files or a packaged .taco connector. Should be the latest version of the connector.
+- Clean logs showing the error. This will be in the "My Tableau Repository\Logs" folder for desktop. For server, the "tsm maintenance ziplogs" command can be used, see this page.
```
Comment (tsjoblad) [inline docs/bug-artifacts.md:17]: Seems redundant with below lines

**Review by tsjoblad — APPROVED** [review]
added comments, otherwise good add!

File: docs/bug-artifacts.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+If you encounter a bug with the Tableau Connector SDk, please file it as a github issue. We will take a look at it and respond as soon as we are able.
+
+# Artifacts
+When filing an bug on github, attaching the following to the issue:
+- A readme file that contains the following (can be in the issue itself):
+    - Your name and company
+    - A short description of the bug
+    - Repro steps with screenshots
+    - The product(s) it appears in
+    - The exact version of the Tableau products you are using (ie Desktop 2020.3.1, Server 2020.2.2, TDVT 2.1.1, etc), as relevant to the bug
```
Comment (rosswbrown) [inline docs/bug-artifacts.md:14]: TDVT and Desktop version are technically independent I believe.  Its probably worth clarifying we need both in the TDVT case.

File: docs/bug-artifacts.md [hunk 4]
```diff
@@ -11,10 +11,11 @@ When filing an bug on github, attaching the following to the issue:
     - A short description of the bug
     - Repro steps with screenshots
     - The product(s) it appears in
-    - The exact version of the Tableau products you are using (ie Desktop 2020.3.1, Server 2020.2.2, TDVT 2.1.1, etc), as relevant to the bug
+    - The exact version of the Tableau products you are using (ie Desktop 2020.3.1, Server 2020.2.2), as relevant to the bug
+    - The exact version of TDVT you are running, if the ug is realted to TDVT or discovered during a TDVt run
```
Comment (rosswbrown) [inline docs/bug-artifacts.md:15]:
```suggestion
    - The exact version of TDVT you are running, if the bug is related to TDVT or discovered during a TDVT run
```

**Review by rosswbrown — APPROVED** [review]

**Pull request merged**